Docs/source/building/summit.rst

```diff
@@ -3,87 +3,106 @@
 Building WarpX on Summit (OLCF)
 ================================
 
-For the `Summit cluster
-<https://www.olcf.ornl.gov/summit/>`__ at OLCF,
-use the following commands to download the source code, and switch to the
-correct branch:
+The `Summit cluster <https://www.olcf.ornl.gov/summit/>`_ is located at OLCF.
 
-::
+If you are new to this system, please see the following resources:
 
-    mkdir warpx_directory
-    cd warpx_directory
+* `Summit user guide <https://docs.olcf.ornl.gov/systems/summit_user_guide.html>`_
+* Batch system: `LSF <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#running-jobs>`_
+* `Production directories <https://docs.olcf.ornl.gov/data/storage_overview.html>`_:
 
-    git clone https://github.com/ECP-WarpX/WarpX.git
-    git clone https://bitbucket.org/berkeleylab/picsar.git
-    git clone --branch development https://github.com/AMReX-Codes/amrex.git
+  * ``$PROJWORK/$proj/``: shared with all members of a project (recommended)
+  * ``$MEMBERWORK/$proj/``: single user (usually smaller quota)
+  * ``$WORLDWORK/$proj/``: shared with all users
+  * Note that the ``$HOME`` directory is mounted as read-only on compute nodes.
+    That means you cannot run in your ``$HOME``.
 
-Then, ``cd`` into the directory ``WarpX`` and use the following set of commands to compile:
 
-::
+Installation
+------------
 
-    module load gcc
-    module load cuda
-    make -j 16 COMP=gcc USE_GPU=TRUE
+We use the following modules and environments on the system.
 
-See :doc:`../running_cpp/platforms` for more information on how to run WarpX on Summit.
+.. code-block:: bash
 
-See :doc:`../visualization/yt` for more information on how to visualize the simulation results.
+   # please set your project account
+   export proj=<yourProject>
 
-.. note::
+   # required dependencies
+   module load gcc/6.4.0
+   module load cuda
 
-   To compile and run WarpX on Summit CPUs, the cuda module is not necessary.
+   # optional: faster re-builds
+   module load ccache
 
-   But to build with the spectral solver, PSATD, you need to use an MPI-enabled version of FFTW like the `fftw` module (because the `cuFFT` tool is not available when cuda is not loaded).
+   # optional: for PSATD support
+   module load fftw
 
-   ::
+   # optional: for QED support
+   module load boost/1.66.0
 
-      module load gcc
-      module load fftw
-      make -j 16 COMP=gcc USE_PSATD=TRUE
+   # optional: for openPMD support
+   module load cmake
+   module load hdf5/1.10.4
+   module load adios2/2.5.0
+   export PKG_CONFIG_PATH=$HOME/sw/openPMD-api-install/lib64/pkgconfig:$PKG_CONFIG_PATH
+   export CMAKE_PREFIX_PATH=$HOME/sw/openPMD-api-install:$CMAKE_PREFIX_PATH
 
+   # optional: just an additional text editor
+   module load nano
 
+   # optional: an alias to request an interactive node for two hours
+   alias getNode="bsub -P $proj -W 2:00 -nnodes 1 -Is /bin/bash"
 
+   # fix system defaults: do not escape $ with a \ on tab completion
+   shopt -s direxpand
```

Comment:
Do these lines belong to the WarpX doc?

Reply:
I think so, everyone I meet is confused why this is the default on this system.